cli/datapipelines_test.go

```diff
@@ -0,0 +1,131 @@
+package cli
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.viam.com/test"
+)
+
+var (
+	mqlString = `[
+		{"$match": { "component_name": "dragino" }},
+		{"$group": {
+			"_id": "$part_id",
+			"count": { "$sum": 1 },
+			"avgTemp": { "$avg": "$data.readings.TempC_SHT" },
+			"avgHum": { "$avg": "$data.readings.Hum_SHT" }
+		}}
+	]`
```

Comment:
Question: We're intentionally supporting json5. Do we want to confirm that it's being parsed as expected? I could see an argument that this would be testing the json5 package, but...

Reply:
is that not covered by comparing to the expected BSON?

Reply:
But stuff like comments?

Reply:
added!